### PR TITLE
PXC-3478 Update ProxySQL installation information (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/howtos/proxysql-v2.rst

### CODE REVIEW

File: doc/source/howtos/proxysql-v2.rst [hunk 1]
```diff
@@ -78,7 +78,35 @@ software, run the corresponding command:
...
+            - For Ubuntu 16.04 ``xenial`` only
+
+The password-based file encryption requires OpenSSL 1.1.1, but Ubuntu 16.04 does not support this OpenSSL version. A special statically linked OpenSSL 1.1.1 binary is packaged with the executable. This packaged binary avoids conflicts with the system OpenSSL and any shared libraries. Each new release rebuilds the binary.
+
+For versions 2.0.13 or lower, extract the files from the archive and change to the directory that contains the extracted files:
+
+   .. code-block:: bash
+      
+      $ # Extract the files (assuming you have changed to the download destination directory)
+      $ tar xzf proxysql-VERSION-Linux-PLATFORM-ARCHITECTURE.tar.gz
```
Comment (rameshvs02) [inline doc/source/howtos/proxysql-v2.rst]: only one binary tarball extract command is required. Please remove this line.

Comment (patrickbirch) [reply doc/source/howtos/proxysql-v2.rst]: Line is removed.

**Comment by rameshvs02** [comment]
LGTM

**Review by rameshvs02 — COMMENTED** [review]
lgtm

**Pull request merged**